cluster/gce/gci/configure-helper.sh

```diff
@@ -455,6 +455,14 @@ function create-master-pki {
     write-pki-data "${CA_KEY}" "${CA_KEY_PATH}"
   fi
```

Comment:
Change back the file permission? (used to be 644)

Reply:
I didn't realize I changed that